plan/plan.go

```diff
@@ -175,3 +188,11 @@ func shouldUpdateTTL(desired, current *endpoint.Endpoint) bool {
 	}
 	return desired.RecordTTL != current.RecordTTL
 }
+
+// validateDNSName checks if the DNS name is correct
+// for now it only removes space and lower case
+func validateDNSName(dnsName string) (string, error) {
```

Comment:
If it doesn't return an `err` and likely won't be in the future, I think we can drop the `err` return which would simplify its usage a lot.